### DISCUSSION TITLE
UseRef and useState gives error for unexpected token

### DISCUSSION THREAD

**Discussion by jahangiramin** [mantinedev/mantine#5164] [Q&A] [answered]
Hi there, I am stuck with some error in below line of code while using Dropzone. I am a newbie and couldn't find any solution on internet so asking here:

const openRef = useRef<() => void>(null);

I get unexpected token error which apparently means the syntax is incorrect. What am I missing here?

same error I get for this line while using Fileupload button as bellow:
const [files, setFiles] = React.useState<FileList | null>();

**Comment by rtivital** [comment, marked as answer]
If you use JavaScript, you need to remove types:
- `const openRef = useRef(null);`
- `const [files, setFiles] = React.useState();`

**Reply by jahangiramin** [reply]
Thankyou so much. so this is due to the typescript syntax and javascript is different. Still learning and this library is amazing. Thumbs up.
